core/settings/base.py

```diff
@@ -148,3 +148,4 @@
 
 # Hashid settings
 HASHID_FIELD_SALT = dotenv.get('HASHID_FIELD_SALT')
+DEBUG = dotenv.get('DEBUG')
```

Comment:
I think setting DEBUG to True makes sense in local.py, i have removed that of production.py